core/src/translate/expr.rs

```diff
@@ -73,21 +73,31 @@ pub fn translate_expr(sql_expr: &SqlExpr) -> Result<Expr> {
             expr,
             negated,
             pattern,
-            escape_char: None,
+            escape_char,
         } => Ok(Expr::Like {
             expr: translate_expr(expr).map(Box::new)?,
             negated: *negated,
             pattern: translate_expr(pattern).map(Box::new)?,
+            escape_char: if let Some(escape) = escape_char {
+                *escape
+            } else {
+                '\\'
+            },
```

Comment:
```rust
escape_char: escape_char.unwrap_or('\\'),
```
It would be good to use `unwrap_or`.
We can apply this to `SqlExpr::ILike`, too.